pkg/api/volume_req.go

```diff
@@ -18,6 +32,6 @@ type VolOptionReq struct {
 
 // VolExpandReq represents a request to expand the volume by adding more bricks
 type VolExpandReq struct {
-	ReplicaCount int      `json:"replica,omitempty"`
-	Bricks       []string `json:"bricks"`
+	ReplicaCount int        `json:"replica,omitempty"`
```

Comment:
This is okay for now. But the expected API to add bricks which aligns with the new create API, is to be able to accept a list of subvols with the new bricks/subvols. For eg.,

```json
[
    {
        "SubvolID": "00adc43c-6eeb-47d9-912a-12daf6cde2c7",
        "Bricks": [
            {
                "NodeID": "8d1c03a7-0ac8-4072-a8c2-83f31c92ccd2",
                "Path": "/new/brick"
            }
        ],
        "Subvols": []
    },
    {
        "SubvolID": "3378e563-705d-4eb0-a9a0-cb9bf2301634",
        "Bricks": [
            {
                "NodeID": "e8986e52-bb46-4bbc-91b6-0c45b0ef1872",
                "Path": "/new/brick"
            }
        ],
        "Subvols": []
    }
]
```

This will make it slightly harder for our cli, but makes the expand api way more powerful.

Reply:
This is for supporting current syntax for Volume Expand. We need to have discussion on all the Sub volume based commands. Like Add new subvol, Delete Subvol, Expand a subvol etc.